### ISSUE TITLE
clipboard windows

### ISSUE THREAD

**Issue by drosan-dev** [jesseduffield/lazygit#828]
**Describe the bug**
*errors.errorString "lipboard"" �� ����� ����७��� ��� ���譥�
��������, �ᯮ��塞�� �ணࠬ��� ��� �������� 䠩���.

**To Reproduce**
ctrl+o on the branch on windows

**Comment by iam-mholle** [comment]
Got the same error, here's the trace:
```text
*errors.errorString 'lipboard"' is not recognized as an internal or external command,
operable program or batch file.
/home/runner/work/lazygit/lazygit/pkg/commands/os.go:186 (0xa3190a)
/home/runner/work/lazygit/lazygit/pkg/commands/os.go:97 (0xa30f09)
/home/runner/work/lazygit/lazygit/pkg/commands/os.go:152 (0xa338e6)
/home/runner/work/lazygit/lazygit/pkg/commands/os.go:466 (0xa338f1)
/home/runner/work/lazygit/lazygit/pkg/gui/commits_panel.go:713 (0xa5228a)
/home/runner/work/lazygit/lazygit/pkg/gui/options_menu_panel.go:52 (0xaa5c29)
/home/runner/work/lazygit/lazygit/pkg/gui/menu_panel.go:92 (0xaa58fa)
/home/runner/work/lazygit/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:896 (0xa1b684)
/home/runner/work/lazygit/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:876 (0xa1b694)
/home/runner/work/lazygit/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:807 (0xa1b438)
/home/runner/work/lazygit/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:497 (0xa19908)
/home/runner/work/lazygit/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:457 (0xa19913)
/home/runner/work/lazygit/lazygit/pkg/gui/gui.go:359 (0xa60291)
/home/runner/work/lazygit/lazygit/pkg/gui/gui.go:368 (0xa60697)
/home/runner/work/lazygit/lazygit/pkg/app/app.go:158 (0xab4122)
/home/runner/work/lazygit/lazygit/main.go:70 (0xab522a)
/opt/hostedtoolcache/go/1.14.2/x64/src/runtime/proc.go:203 (0x438912)
/opt/hostedtoolcache/go/1.14.2/x64/src/runtime/asm_amd64.s:1373 (0x463421)
```

**Comment by iam-mholle** [comment]
> 
> 
> @drosan-dev @iam-mholle I'm not able to test on windows right now but I've noticed an issue with the command. Would you be able to see if the issue is resolved in #958?

Hi @jesseduffield

I just tried #958, but unfortunately the "copy commit SHA to clipboard" action still results in a crash on windows. Here's the output:
```
2020/08/24 18:56:39 An error occurred! Please create an issue at https://github.com/jesseduffield/lazygit/issues

*errors.errorString The system cannot find the path specified.

D:/Projects/lazygit/pkg/commands/os.go:203 (0xa690f8)
        sanitisedCommandOutput: return outputString, errors.New(outputString)
D:/Projects/lazygit/pkg/commands/os.go:99 (0xa68405)
        (*OSCommand).RunCommandWithOutput: return sanitisedCommandOutput(cmd.CombinedOutput())
D:/Projects/lazygit/pkg/commands/os.go:169 (0xa6b23b)
        (*OSCommand).RunCommand: _, err := c.RunCommandWithOutput(formatString, formatArgs...)
D:/Projects/lazygit/pkg/commands/os.go:485 (0xa6b246)
        (*OSCommand).CopyToClipboard: return c.RunCommand(command)
D:/Projects/lazygit/pkg/gui/global_handlers.go:195 (0xa9707e)
        (*Gui).handleCopySelectedSideContextItemToClipboard: return gui.OSCommand.CopyToClipboard(itemId)
D:/Projects/lazygit/pkg/gui/view_helpers.go:353 (0xae96ad)
        (*Gui).wrappedHandler.func1: return f()
D:/Projects/lazygit/pkg/gui/options_menu_panel.go:52 (0xae46ac)
        (*Gui).handleCreateOptionsMenu.func1: return innerBinding.Handler(g, v)
D:/Projects/lazygit/pkg/gui/menu_panel.go:99 (0xac115f)
        (*Gui).onMenuPress: if err := gui.State.MenuItems[selectedLine].onPress(); err != nil {
D:/Projects/lazygit/pkg/gui/view_helpers.go:353 (0xae96ad)
        (*Gui).wrappedHandler.func1: return f()
D:/Projects/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:898 (0xa4b911)
        (*Gui).execKeybinding: if err := kb.handler(g, v); err != nil {
D:/Projects/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:878 (0xa4b926)
        (*Gui).execKeybindings: return g.execKeybinding(v, kb)
D:/Projects/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:809 (0xa4b69d)
        (*Gui).onKey: matched, err := g.execKeybindings(g.currentView, ev)
D:/Projects/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:497 (0xa49a2a)
        (*Gui).handleEvent: return g.onKey(ev)
D:/Projects/lazygit/vendor/github.com/jesseduffield/gocui/gui.go:457 (0xa49a35)
        (*Gui).MainLoop: if err := g.handleEvent(&ev); err != nil {
D:/Projects/lazygit/pkg/gui/gui.go:447 (0xa9900c)
        (*Gui).Run: err = g.MainLoop()
D:/Projects/lazygit/pkg/gui/gui.go:456 (0xa99307)
        (*Gui).RunWithSubprocesses: if err := gui.Run(); err != nil {
D:/Projects/lazygit/pkg/app/app.go:179 (0xaf4f93)
        (*App).Run: err := app.Gui.RunWithSubprocesses()
D:/Projects/lazygit/main.go:70 (0xaf6073)
        main: err = app.Run()
c:/go/src/runtime/proc.go:204 (0x44b269)
        main: fn()
c:/go/src/runtime/asm_amd64.s:1374 (0x47aea1)
        goexit: BYTE    $0x90   // NOP
```
